roles/ceph-common/tasks/main.yml

```diff
@@ -200,7 +208,9 @@
   register: cluster_uuid
   become: false
   always_run: true
-  when: generate_fsid
+  when:
+    - generate_fsid
+    - ceph_current_fsid.rc != 0
```

Comment:
Wouldn't you still want to run this task to register `cluster_uuid`? It looks like `cluster_uuid` really isn't used later on, but that doesn't mean it won't ever get used in the future.